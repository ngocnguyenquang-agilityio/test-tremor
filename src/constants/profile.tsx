import { MdEmail, MdHome, MdSettings } from "react-icons/md";

export const TABS_HEADER = [
  { name: "App", icon: <MdHome /> },
  { name: "Message", icon: <MdEmail /> },
  { name: "Setting", icon: <MdSettings /> },
];
