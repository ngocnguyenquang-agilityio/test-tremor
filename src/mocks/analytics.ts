export const ANALYTIC_SALES_CARD = [
  {
    flag: "/assets/images/flags/US.webp",
    country: "United State",
    sales: "2500",
    value: "$230,900",
    bounce: "29.9%",
  },
  {
    flag: "/assets/images/flags/DE.webp",
    country: "Germany",
    sales: "3.900",
    value: "$440,000",
    bounce: "40.22%",
  },
  {
    flag: "/assets/images/flags/GB.webp",
    country: "Great Britain",
    sales: "1.400",
    value: "$190,700",
    bounce: "23.44%",
  },
  {
    flag: "/assets/images/flags/BR.webp",
    country: "Brasil",
    sales: "562",
    value: "$143,960",
    bounce: "32.14%",
  },
];

export const STATISTICAL_DATA = [
  {
    type: "Bookings",
    amount: "281",
    totalAmount: "55%",
    duration: "than last week",
  },
  {
    type: "Today's Users",
    amount: "2,300",
    totalAmount: "3%",
    duration: "than last month",
  },
  {
    type: "Revenue",
    amount: "34k",
    totalAmount: "1%",
    duration: "than yesterday",
  },
  {
    type: "Followers",
    amount: "+91",
    totalAmount: "",
    duration: "Just updated",
  },
];
