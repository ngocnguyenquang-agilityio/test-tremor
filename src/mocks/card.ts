export const PROFILE_INFO = {
  description:
    "Hi, I’m Alec Thompson, Decisions: If you can’t decide, the answer is no. If two equally difficult paths, choose the one more painful in the short term (pain avoidance is creating an illusion of equality)",
  info: [
    {
      key: "Full Name",
      value: "Alec M. Thompson",
    },
    {
      key: "Mobile",
      value: "(44) 123 1234 123",
    },
    {
      key: "Email",
      value: "alecthompson@mail.com",
    },
    {
      key: "Location",
      value: "USA",
    },
  ],
};
