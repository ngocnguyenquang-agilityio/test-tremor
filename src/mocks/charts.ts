export const LINE_CHART_DATA = [
  {
    id: "sale",
    display: "Daily Sales",
    descValue: "(+15%)",
    desc: "increase in today sales.",
    modified: "updated 4 min ago",
    data: [
      {
        date: "Apr",
        "Desktop apps": 50,
      },
      {
        date: "May",
        "Desktop apps": 40,
      },
      {
        date: "Jun",
        "Desktop apps": 300,
      },
      {
        date: "Jul",
        "Desktop apps": 220,
      },
      {
        date: "Aug",
        "Desktop apps": 500,
      },
      {
        date: "Sep",
        "Desktop apps": 250,
      },
      {
        date: "Oct",
        "Desktop apps": 400,
      },
      {
        date: "Nov",
        "Desktop apps": 230,
      },
      {
        date: "Dec",
        "Desktop apps": 500,
      },
    ],
  },
  {
    id: "performance",
    display: "Completed Tasks",
    descValue: "(+15%)",
    desc: "Last Campaign Performance",
    modified: "just updated",
    data: [
      {
        date: "Apr",
        "Mobile apps": 50,
      },
      {
        date: "May",
        "Mobile apps": 40,
      },
      {
        date: "Jun",
        "Mobile apps": 300,
      },
      {
        date: "Jul",
        "Mobile apps": 320,
      },
      {
        date: "Aug",
        "Mobile apps": 500,
      },
      {
        date: "Sep",
        "Mobile apps": 350,
      },
      {
        date: "Oct",
        "Mobile apps": 200,
      },
      {
        date: "Nov",
        "Mobile apps": 230,
      },
      {
        date: "Dec",
        "Mobile apps": 500,
      },
    ],
  },
];
