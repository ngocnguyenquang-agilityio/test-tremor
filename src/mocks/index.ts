export * from "./card";
export * from "./profile";
export * from "./project";
export * from "./charts";
export * from "./analytics";
