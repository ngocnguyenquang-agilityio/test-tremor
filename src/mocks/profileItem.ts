export const PROFILE_ITEM = {
  src: "/images/avatar/avatar-md.webp",
  alt: "image",
  name: "Sophie.B",
  description: "Hi! I need more information..",
};
