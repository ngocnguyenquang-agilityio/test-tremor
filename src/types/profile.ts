export type Profile = {
  src: string;
  alt: string;
  name: string;
  description: string;
};
