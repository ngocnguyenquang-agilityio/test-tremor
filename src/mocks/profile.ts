import { ConversationHistory } from "@/types/profile";

export const PROFILE_HEADER = {
  avatarUrl: "/images/avatar/avatar-lg.webp",
  alt: "image",
  name: "Richard Davis",
  role: "CEO / Co-Founder",
};

export const PROFILE_ITEM = {
  id: 1,
  avatar: "/images/avatar/avatar-md.webp",
  name: "Sophie.B",
  lastConversation: "Hi! I need more information..",
};

export const PROFILE_CONVERSATIONS: ConversationHistory[] = [
  {
    id: 1,
    avatar: "/images/avatar/avatar-md.webp",
    name: "Anne Marie",
    lastConversation: "Awesome work, can you..",
  },
  {
    id: 2,
    avatar: "/images/avatar/avatar-lg.webp",
    name: "Ivanna",
    lastConversation: "About files I can..",
  },
  {
    id: 3,
    avatar: "/images/avatar/avatar-xs.webp",
    name: "Peterson",
    lastConversation: "Have a great afternoon..",
  },
  {
    id: 4,
    avatar: "/images/avatar/avatar-md.webp",
    name: "Sophie.B",
    lastConversation: "Hi! I need more information..",
  },
  {
    id: 5,
    avatar: "/images/avatar/avatar-sm.webp",
    name: "Nick Daniel",
    lastConversation: "Hi! I need more information..",
  },
];

export const PROFILE_INFO_PROJECT_CARD = [
  {
    id: 1,
    cover: "/assets/images/profile-project.webp",
    participants: [
      { avatar: "/images/avatar/avatar-xs.webp" },
      { avatar: "/images/avatar/avatar-xs.webp" },
    ],
    primaryName: "modern",
    description:
      "As Uber works through a huge amount of internal management turmoil.",
    secondaryName: "#1",
  },
  {
    id: 2,
    cover: "/assets/images/profile-project.webp",
    participants: [
      { avatar: "/images/avatar/avatar-xs.webp" },
      { avatar: "/images/avatar/avatar-xs.webp" },
    ],
    primaryName: "scandinavian",
    description:
      "As Uber works through a huge amount of internal management turmoil.",
    secondaryName: "#2",
  },
  {
    id: 3,
    cover: "/assets/images/profile-project.webp",
    participants: [
      { avatar: "/images/avatar/avatar-xs.webp" },
      { avatar: "/images/avatar/avatar-xs.webp" },
    ],
    primaryName: "minimalist",
    description:
      "Different people have different taste, and various types of music.",
    secondaryName: "#2",
  },
  {
    id: 4,
    cover: "/assets/images/profile-project.webp",
    participants: [
      { avatar: "/images/avatar/avatar-xs.webp" },
      { avatar: "/images/avatar/avatar-xs.webp" },
    ],
    primaryName: "gothic",
    description:
      "Why would anyone pick blue over pink? Pink is obviously a better color.",
    secondaryName: "#4",
  },
];
