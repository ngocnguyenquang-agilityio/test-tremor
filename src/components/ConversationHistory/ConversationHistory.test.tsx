import { render } from "@testing-library/react";

// Components
import { ConversationHistory } from "..";

// Mocks
import { PROFILE_CONVERSATIONS } from "@/mocks/profile";

describe("ConversationHistory component", () => {
  const props = {
    profileList: PROFILE_CONVERSATIONS,
    onClick: jest.fn(),
  };

  const profile = () => {
    return render(<ConversationHistory {...props} />);
  };

  it("Should render ConversationHistory snapshot correctly", () => {
    expect(profile()).toMatchSnapshot();
  });
});
