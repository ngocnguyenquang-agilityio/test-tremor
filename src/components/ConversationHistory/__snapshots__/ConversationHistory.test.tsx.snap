// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConversationHistory component Should render ConversationHistory snapshot correctly 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <p
        class="dark:text-dark-tremor-content text-tremor-title leading-relaxed font-bold tracking-[0.0075em] opacity-100 capitalize no-underline text-primary py-4"
      >
        Profile Conversations
      </p>
      <div
        class="tremor-Flex-root flex w-full flex-row justify-between items-center mb-2 py-2 flex-wrap xs:flex-nowrap"
      >
        <div
          class="tremor-Flex-root flex w-full flex-row items-center justify-start flex-wrap sm:flex-nowrap"
        >
          <div
            class="w-12 h-12"
          >
            <div
              class="inline-flex rounded-full overflow-hidden shadow-md"
            >
              <img
                alt="Anne Marie"
                data-nimg="1"
                decoding="async"
                fetchpriority="high"
                height="48"
                src="/_next/image?url=%2Fimages%2Favatar%2Favatar-md.webp&w=96&q=75"
                srcset="/_next/image?url=%2Fimages%2Favatar%2Favatar-md.webp&w=48&q=75 1x, /_next/image?url=%2Fimages%2Favatar%2Favatar-md.webp&w=96&q=75 2x"
                style="color: transparent;"
                width="48"
              />
            </div>
          </div>
          <div
            class="max-w-[180px] font-medium dark:text-white pl-4"
          >
            <p
              class="text-tremor-default dark:text-dark-tremor-content text-sm text-primary font-semibold"
            >
              Anne Marie
            </p>
            <p
              class="text-tremor-default dark:text-dark-tremor-content text-xs text-secondary font-light truncate"
            >
              Awesome work, can you..
            </p>
          </div>
        </div>
        <button
          class="tremor-Button-root flex-shrink-0 inline-flex justify-center items-center group font-medium outline-none rounded-tremor-default shadow-tremor-input dark:shadow-dark-tremor-input px-4 py-2 text-sm dark:bg-dark-tremor-brand border-tremor-brand dark:border-dark-tremor-brand text-tremor-brand-inverted dark:text-dark-tremor-brand-inverted dark:hover:bg-dark-tremor-brand-emphasis hover:border-tremor-brand-emphasis dark:hover:border-dark-tremor-brand-emphasis min-w-[64px] m-0 border-0 hover:bg-[none] bg-transparent"
        >
          <span
            class="tremor-Button-text text-sm whitespace-nowrap"
          >
            <p
              class="text-tremor-default dark:text-dark-tremor-content text-primary text-xs text-center font-bold uppercase leading-[17px]"
            >
              Reply
            </p>
          </span>
        </button>
      </div>
      <div
        class="tremor-Flex-root flex w-full flex-row justify-between items-center mb-2 py-2 flex-wrap xs:flex-nowrap"
      >
        <div
          class="tremor-Flex-root flex w-full flex-row items-center justify-start flex-wrap sm:flex-nowrap"
        >
          <div
            class="w-12 h-12"
          >
            <div
              class="inline-flex rounded-full overflow-hidden shadow-md"
            >
              <img
                alt="Ivanna"
                data-nimg="1"
                decoding="async"
                fetchpriority="high"
                height="48"
                src="/_next/image?url=%2Fimages%2Favatar%2Favatar-lg.webp&w=96&q=75"
                srcset="/_next/image?url=%2Fimages%2Favatar%2Favatar-lg.webp&w=48&q=75 1x, /_next/image?url=%2Fimages%2Favatar%2Favatar-lg.webp&w=96&q=75 2x"
                style="color: transparent;"
                width="48"
              />
            </div>
          </div>
          <div
            class="max-w-[180px] font-medium dark:text-white pl-4"
          >
            <p
              class="text-tremor-default dark:text-dark-tremor-content text-sm text-primary font-semibold"
            >
              Ivanna
            </p>
            <p
              class="text-tremor-default dark:text-dark-tremor-content text-xs text-secondary font-light truncate"
            >
              About files I can..
            </p>
          </div>
        </div>
        <button
          class="tremor-Button-root flex-shrink-0 inline-flex justify-center items-center group font-medium outline-none rounded-tremor-default shadow-tremor-input dark:shadow-dark-tremor-input px-4 py-2 text-sm dark:bg-dark-tremor-brand border-tremor-brand dark:border-dark-tremor-brand text-tremor-brand-inverted dark:text-dark-tremor-brand-inverted dark:hover:bg-dark-tremor-brand-emphasis hover:border-tremor-brand-emphasis dark:hover:border-dark-tremor-brand-emphasis min-w-[64px] m-0 border-0 hover:bg-[none] bg-transparent"
        >
          <span
            class="tremor-Button-text text-sm whitespace-nowrap"
          >
            <p
              class="text-tremor-default dark:text-dark-tremor-content text-primary text-xs text-center font-bold uppercase leading-[17px]"
            >
              Reply
            </p>
          </span>
        </button>
      </div>
      <div
        class="tremor-Flex-root flex w-full flex-row justify-between items-center mb-2 py-2 flex-wrap xs:flex-nowrap"
      >
        <div
          class="tremor-Flex-root flex w-full flex-row items-center justify-start flex-wrap sm:flex-nowrap"
        >
          <div
            class="w-12 h-12"
          >
            <div
              class="inline-flex rounded-full overflow-hidden shadow-md"
            >
              <img
                alt="Peterson"
                data-nimg="1"
                decoding="async"
                fetchpriority="high"
                height="48"
                src="/_next/image?url=%2Fimages%2Favatar%2Favatar-xs.webp&w=96&q=75"
                srcset="/_next/image?url=%2Fimages%2Favatar%2Favatar-xs.webp&w=48&q=75 1x, /_next/image?url=%2Fimages%2Favatar%2Favatar-xs.webp&w=96&q=75 2x"
                style="color: transparent;"
                width="48"
              />
            </div>
          </div>
          <div
            class="max-w-[180px] font-medium dark:text-white pl-4"
          >
            <p
              class="text-tremor-default dark:text-dark-tremor-content text-sm text-primary font-semibold"
            >
              Peterson
            </p>
            <p
              class="text-tremor-default dark:text-dark-tremor-content text-xs text-secondary font-light truncate"
            >
              Have a great afternoon..
            </p>
          </div>
        </div>
        <button
          class="tremor-Button-root flex-shrink-0 inline-flex justify-center items-center group font-medium outline-none rounded-tremor-default shadow-tremor-input dark:shadow-dark-tremor-input px-4 py-2 text-sm dark:bg-dark-tremor-brand border-tremor-brand dark:border-dark-tremor-brand text-tremor-brand-inverted dark:text-dark-tremor-brand-inverted dark:hover:bg-dark-tremor-brand-emphasis hover:border-tremor-brand-emphasis dark:hover:border-dark-tremor-brand-emphasis min-w-[64px] m-0 border-0 hover:bg-[none] bg-transparent"
        >
          <span
            class="tremor-Button-text text-sm whitespace-nowrap"
          >
            <p
              class="text-tremor-default dark:text-dark-tremor-content text-primary text-xs text-center font-bold uppercase leading-[17px]"
            >
              Reply
            </p>
          </span>
        </button>
      </div>
      <div
        class="tremor-Flex-root flex w-full flex-row justify-between items-center mb-2 py-2 flex-wrap xs:flex-nowrap"
      >
        <div
          class="tremor-Flex-root flex w-full flex-row items-center justify-start flex-wrap sm:flex-nowrap"
        >
          <div
            class="w-12 h-12"
          >
            <div
              class="inline-flex rounded-full overflow-hidden shadow-md"
            >
              <img
                alt="Sophie.B"
                data-nimg="1"
                decoding="async"
                fetchpriority="high"
                height="48"
                src="/_next/image?url=%2Fimages%2Favatar%2Favatar-md.webp&w=96&q=75"
                srcset="/_next/image?url=%2Fimages%2Favatar%2Favatar-md.webp&w=48&q=75 1x, /_next/image?url=%2Fimages%2Favatar%2Favatar-md.webp&w=96&q=75 2x"
                style="color: transparent;"
                width="48"
              />
            </div>
          </div>
          <div
            class="max-w-[180px] font-medium dark:text-white pl-4"
          >
            <p
              class="text-tremor-default dark:text-dark-tremor-content text-sm text-primary font-semibold"
            >
              Sophie.B
            </p>
            <p
              class="text-tremor-default dark:text-dark-tremor-content text-xs text-secondary font-light truncate"
            >
              Hi! I need more information..
            </p>
          </div>
        </div>
        <button
          class="tremor-Button-root flex-shrink-0 inline-flex justify-center items-center group font-medium outline-none rounded-tremor-default shadow-tremor-input dark:shadow-dark-tremor-input px-4 py-2 text-sm dark:bg-dark-tremor-brand border-tremor-brand dark:border-dark-tremor-brand text-tremor-brand-inverted dark:text-dark-tremor-brand-inverted dark:hover:bg-dark-tremor-brand-emphasis hover:border-tremor-brand-emphasis dark:hover:border-dark-tremor-brand-emphasis min-w-[64px] m-0 border-0 hover:bg-[none] bg-transparent"
        >
          <span
            class="tremor-Button-text text-sm whitespace-nowrap"
          >
            <p
              class="text-tremor-default dark:text-dark-tremor-content text-primary text-xs text-center font-bold uppercase leading-[17px]"
            >
              Reply
            </p>
          </span>
        </button>
      </div>
      <div
        class="tremor-Flex-root flex w-full flex-row justify-between items-center mb-2 py-2 flex-wrap xs:flex-nowrap"
      >
        <div
          class="tremor-Flex-root flex w-full flex-row items-center justify-start flex-wrap sm:flex-nowrap"
        >
          <div
            class="w-12 h-12"
          >
            <div
              class="inline-flex rounded-full overflow-hidden shadow-md"
            >
              <img
                alt="Nick Daniel"
                data-nimg="1"
                decoding="async"
                fetchpriority="high"
                height="48"
                src="/_next/image?url=%2Fimages%2Favatar%2Favatar-sm.webp&w=96&q=75"
                srcset="/_next/image?url=%2Fimages%2Favatar%2Favatar-sm.webp&w=48&q=75 1x, /_next/image?url=%2Fimages%2Favatar%2Favatar-sm.webp&w=96&q=75 2x"
                style="color: transparent;"
                width="48"
              />
            </div>
          </div>
          <div
            class="max-w-[180px] font-medium dark:text-white pl-4"
          >
            <p
              class="text-tremor-default dark:text-dark-tremor-content text-sm text-primary font-semibold"
            >
              Nick Daniel
            </p>
            <p
              class="text-tremor-default dark:text-dark-tremor-content text-xs text-secondary font-light truncate"
            >
              Hi! I need more information..
            </p>
          </div>
        </div>
        <button
          class="tremor-Button-root flex-shrink-0 inline-flex justify-center items-center group font-medium outline-none rounded-tremor-default shadow-tremor-input dark:shadow-dark-tremor-input px-4 py-2 text-sm dark:bg-dark-tremor-brand border-tremor-brand dark:border-dark-tremor-brand text-tremor-brand-inverted dark:text-dark-tremor-brand-inverted dark:hover:bg-dark-tremor-brand-emphasis hover:border-tremor-brand-emphasis dark:hover:border-dark-tremor-brand-emphasis min-w-[64px] m-0 border-0 hover:bg-[none] bg-transparent"
        >
          <span
            class="tremor-Button-text text-sm whitespace-nowrap"
          >
            <p
              class="text-tremor-default dark:text-dark-tremor-content text-primary text-xs text-center font-bold uppercase leading-[17px]"
            >
              Reply
            </p>
          </span>
        </button>
      </div>
    </div>
  </body>,
  "container": <div>
    <p
      class="dark:text-dark-tremor-content text-tremor-title leading-relaxed font-bold tracking-[0.0075em] opacity-100 capitalize no-underline text-primary py-4"
    >
      Profile Conversations
    </p>
    <div
      class="tremor-Flex-root flex w-full flex-row justify-between items-center mb-2 py-2 flex-wrap xs:flex-nowrap"
    >
      <div
        class="tremor-Flex-root flex w-full flex-row items-center justify-start flex-wrap sm:flex-nowrap"
      >
        <div
          class="w-12 h-12"
        >
          <div
            class="inline-flex rounded-full overflow-hidden shadow-md"
          >
            <img
              alt="Anne Marie"
              data-nimg="1"
              decoding="async"
              fetchpriority="high"
              height="48"
              src="/_next/image?url=%2Fimages%2Favatar%2Favatar-md.webp&w=96&q=75"
              srcset="/_next/image?url=%2Fimages%2Favatar%2Favatar-md.webp&w=48&q=75 1x, /_next/image?url=%2Fimages%2Favatar%2Favatar-md.webp&w=96&q=75 2x"
              style="color: transparent;"
              width="48"
            />
          </div>
        </div>
        <div
          class="max-w-[180px] font-medium dark:text-white pl-4"
        >
          <p
            class="text-tremor-default dark:text-dark-tremor-content text-sm text-primary font-semibold"
          >
            Anne Marie
          </p>
          <p
            class="text-tremor-default dark:text-dark-tremor-content text-xs text-secondary font-light truncate"
          >
            Awesome work, can you..
          </p>
        </div>
      </div>
      <button
        class="tremor-Button-root flex-shrink-0 inline-flex justify-center items-center group font-medium outline-none rounded-tremor-default shadow-tremor-input dark:shadow-dark-tremor-input px-4 py-2 text-sm dark:bg-dark-tremor-brand border-tremor-brand dark:border-dark-tremor-brand text-tremor-brand-inverted dark:text-dark-tremor-brand-inverted dark:hover:bg-dark-tremor-brand-emphasis hover:border-tremor-brand-emphasis dark:hover:border-dark-tremor-brand-emphasis min-w-[64px] m-0 border-0 hover:bg-[none] bg-transparent"
      >
        <span
          class="tremor-Button-text text-sm whitespace-nowrap"
        >
          <p
            class="text-tremor-default dark:text-dark-tremor-content text-primary text-xs text-center font-bold uppercase leading-[17px]"
          >
            Reply
          </p>
        </span>
      </button>
    </div>
    <div
      class="tremor-Flex-root flex w-full flex-row justify-between items-center mb-2 py-2 flex-wrap xs:flex-nowrap"
    >
      <div
        class="tremor-Flex-root flex w-full flex-row items-center justify-start flex-wrap sm:flex-nowrap"
      >
        <div
          class="w-12 h-12"
        >
          <div
            class="inline-flex rounded-full overflow-hidden shadow-md"
          >
            <img
              alt="Ivanna"
              data-nimg="1"
              decoding="async"
              fetchpriority="high"
              height="48"
              src="/_next/image?url=%2Fimages%2Favatar%2Favatar-lg.webp&w=96&q=75"
              srcset="/_next/image?url=%2Fimages%2Favatar%2Favatar-lg.webp&w=48&q=75 1x, /_next/image?url=%2Fimages%2Favatar%2Favatar-lg.webp&w=96&q=75 2x"
              style="color: transparent;"
              width="48"
            />
          </div>
        </div>
        <div
          class="max-w-[180px] font-medium dark:text-white pl-4"
        >
          <p
            class="text-tremor-default dark:text-dark-tremor-content text-sm text-primary font-semibold"
          >
            Ivanna
          </p>
          <p
            class="text-tremor-default dark:text-dark-tremor-content text-xs text-secondary font-light truncate"
          >
            About files I can..
          </p>
        </div>
      </div>
      <button
        class="tremor-Button-root flex-shrink-0 inline-flex justify-center items-center group font-medium outline-none rounded-tremor-default shadow-tremor-input dark:shadow-dark-tremor-input px-4 py-2 text-sm dark:bg-dark-tremor-brand border-tremor-brand dark:border-dark-tremor-brand text-tremor-brand-inverted dark:text-dark-tremor-brand-inverted dark:hover:bg-dark-tremor-brand-emphasis hover:border-tremor-brand-emphasis dark:hover:border-dark-tremor-brand-emphasis min-w-[64px] m-0 border-0 hover:bg-[none] bg-transparent"
      >
        <span
          class="tremor-Button-text text-sm whitespace-nowrap"
        >
          <p
            class="text-tremor-default dark:text-dark-tremor-content text-primary text-xs text-center font-bold uppercase leading-[17px]"
          >
            Reply
          </p>
        </span>
      </button>
    </div>
    <div
      class="tremor-Flex-root flex w-full flex-row justify-between items-center mb-2 py-2 flex-wrap xs:flex-nowrap"
    >
      <div
        class="tremor-Flex-root flex w-full flex-row items-center justify-start flex-wrap sm:flex-nowrap"
      >
        <div
          class="w-12 h-12"
        >
          <div
            class="inline-flex rounded-full overflow-hidden shadow-md"
          >
            <img
              alt="Peterson"
              data-nimg="1"
              decoding="async"
              fetchpriority="high"
              height="48"
              src="/_next/image?url=%2Fimages%2Favatar%2Favatar-xs.webp&w=96&q=75"
              srcset="/_next/image?url=%2Fimages%2Favatar%2Favatar-xs.webp&w=48&q=75 1x, /_next/image?url=%2Fimages%2Favatar%2Favatar-xs.webp&w=96&q=75 2x"
              style="color: transparent;"
              width="48"
            />
          </div>
        </div>
        <div
          class="max-w-[180px] font-medium dark:text-white pl-4"
        >
          <p
            class="text-tremor-default dark:text-dark-tremor-content text-sm text-primary font-semibold"
          >
            Peterson
          </p>
          <p
            class="text-tremor-default dark:text-dark-tremor-content text-xs text-secondary font-light truncate"
          >
            Have a great afternoon..
          </p>
        </div>
      </div>
      <button
        class="tremor-Button-root flex-shrink-0 inline-flex justify-center items-center group font-medium outline-none rounded-tremor-default shadow-tremor-input dark:shadow-dark-tremor-input px-4 py-2 text-sm dark:bg-dark-tremor-brand border-tremor-brand dark:border-dark-tremor-brand text-tremor-brand-inverted dark:text-dark-tremor-brand-inverted dark:hover:bg-dark-tremor-brand-emphasis hover:border-tremor-brand-emphasis dark:hover:border-dark-tremor-brand-emphasis min-w-[64px] m-0 border-0 hover:bg-[none] bg-transparent"
      >
        <span
          class="tremor-Button-text text-sm whitespace-nowrap"
        >
          <p
            class="text-tremor-default dark:text-dark-tremor-content text-primary text-xs text-center font-bold uppercase leading-[17px]"
          >
            Reply
          </p>
        </span>
      </button>
    </div>
    <div
      class="tremor-Flex-root flex w-full flex-row justify-between items-center mb-2 py-2 flex-wrap xs:flex-nowrap"
    >
      <div
        class="tremor-Flex-root flex w-full flex-row items-center justify-start flex-wrap sm:flex-nowrap"
      >
        <div
          class="w-12 h-12"
        >
          <div
            class="inline-flex rounded-full overflow-hidden shadow-md"
          >
            <img
              alt="Sophie.B"
              data-nimg="1"
              decoding="async"
              fetchpriority="high"
              height="48"
              src="/_next/image?url=%2Fimages%2Favatar%2Favatar-md.webp&w=96&q=75"
              srcset="/_next/image?url=%2Fimages%2Favatar%2Favatar-md.webp&w=48&q=75 1x, /_next/image?url=%2Fimages%2Favatar%2Favatar-md.webp&w=96&q=75 2x"
              style="color: transparent;"
              width="48"
            />
          </div>
        </div>
        <div
          class="max-w-[180px] font-medium dark:text-white pl-4"
        >
          <p
            class="text-tremor-default dark:text-dark-tremor-content text-sm text-primary font-semibold"
          >
            Sophie.B
          </p>
          <p
            class="text-tremor-default dark:text-dark-tremor-content text-xs text-secondary font-light truncate"
          >
            Hi! I need more information..
          </p>
        </div>
      </div>
      <button
        class="tremor-Button-root flex-shrink-0 inline-flex justify-center items-center group font-medium outline-none rounded-tremor-default shadow-tremor-input dark:shadow-dark-tremor-input px-4 py-2 text-sm dark:bg-dark-tremor-brand border-tremor-brand dark:border-dark-tremor-brand text-tremor-brand-inverted dark:text-dark-tremor-brand-inverted dark:hover:bg-dark-tremor-brand-emphasis hover:border-tremor-brand-emphasis dark:hover:border-dark-tremor-brand-emphasis min-w-[64px] m-0 border-0 hover:bg-[none] bg-transparent"
      >
        <span
          class="tremor-Button-text text-sm whitespace-nowrap"
        >
          <p
            class="text-tremor-default dark:text-dark-tremor-content text-primary text-xs text-center font-bold uppercase leading-[17px]"
          >
            Reply
          </p>
        </span>
      </button>
    </div>
    <div
      class="tremor-Flex-root flex w-full flex-row justify-between items-center mb-2 py-2 flex-wrap xs:flex-nowrap"
    >
      <div
        class="tremor-Flex-root flex w-full flex-row items-center justify-start flex-wrap sm:flex-nowrap"
      >
        <div
          class="w-12 h-12"
        >
          <div
            class="inline-flex rounded-full overflow-hidden shadow-md"
          >
            <img
              alt="Nick Daniel"
              data-nimg="1"
              decoding="async"
              fetchpriority="high"
              height="48"
              src="/_next/image?url=%2Fimages%2Favatar%2Favatar-sm.webp&w=96&q=75"
              srcset="/_next/image?url=%2Fimages%2Favatar%2Favatar-sm.webp&w=48&q=75 1x, /_next/image?url=%2Fimages%2Favatar%2Favatar-sm.webp&w=96&q=75 2x"
              style="color: transparent;"
              width="48"
            />
          </div>
        </div>
        <div
          class="max-w-[180px] font-medium dark:text-white pl-4"
        >
          <p
            class="text-tremor-default dark:text-dark-tremor-content text-sm text-primary font-semibold"
          >
            Nick Daniel
          </p>
          <p
            class="text-tremor-default dark:text-dark-tremor-content text-xs text-secondary font-light truncate"
          >
            Hi! I need more information..
          </p>
        </div>
      </div>
      <button
        class="tremor-Button-root flex-shrink-0 inline-flex justify-center items-center group font-medium outline-none rounded-tremor-default shadow-tremor-input dark:shadow-dark-tremor-input px-4 py-2 text-sm dark:bg-dark-tremor-brand border-tremor-brand dark:border-dark-tremor-brand text-tremor-brand-inverted dark:text-dark-tremor-brand-inverted dark:hover:bg-dark-tremor-brand-emphasis hover:border-tremor-brand-emphasis dark:hover:border-dark-tremor-brand-emphasis min-w-[64px] m-0 border-0 hover:bg-[none] bg-transparent"
      >
        <span
          class="tremor-Button-text text-sm whitespace-nowrap"
        >
          <p
            class="text-tremor-default dark:text-dark-tremor-content text-primary text-xs text-center font-bold uppercase leading-[17px]"
          >
            Reply
          </p>
        </span>
      </button>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
